docs/config.json

```diff
@@ -74,7 +74,7 @@
       "pages": [
         {
           "title": "DocSearch Technical Overview (the Scraper)",
-          "url": "scraper-overview.html"
+          "url": "crawler-overview.html"
```

Comment:
don't you want to change the page's name instead?

Reply:
I think it should be enough

Reply:
i mean changing the file instead of the url in the config :)

Reply:
Let’s keep it this way